Taskfile.yml

```diff
@@ -7,7 +7,7 @@ tasks:
 
   work:
     cmds:
-      - cmd: go work init . ./core ./parse
+      - cmd: go work init . ./core
```

Comment:
We have `./test` again

Reply:
Should also update test/go.mod:

```diff
diff --git a/test/go.mod b/test/go.mod
index d0f52116..e1bd7756 100644
--- a/test/go.mod
+++ b/test/go.mod
@@ -5,7 +5,6 @@ go 1.22.7
 replace (
        github.com/kwilteam/kwil-db => ../
        github.com/kwilteam/kwil-db/core => ../core
-       github.com/kwilteam/kwil-db/parse => ../parse
 )
 
 require (
@@ -14,7 +13,6 @@ require (
        github.com/ethereum/go-ethereum v1.14.12
        github.com/kwilteam/kwil-db v0.0.0-00010101000000-000000000000
        github.com/kwilteam/kwil-db/core v0.3.1-0.20241212163115-7353f2761884
-       github.com/kwilteam/kwil-db/parse v0.3.1-0.20241212163115-7353f2761884
        github.com/stretchr/testify v1.10.0
        github.com/testcontainers/testcontainers-go v0.34.0
        github.com/testcontainers/testcontainers-go/modules/compose v0.34.0
```

Reply:
I don't have any go.mods in my `test` folder. Mine is totally empty:

```
➜  kwil-db git:(v0.10-sql-nodev2) ✗ ls test
acceptance  integration
➜  kwil-db git:(v0.10-sql-nodev2) ✗ ls test/acceptance 
➜  kwil-db git:(v0.10-sql-nodev2) ✗ ls test/integration 
➜  kwil-db git:(v0.10-sql-nodev2) ✗ 
```